### PR TITLE
Integrating accessibilityLabel functionality for MenuButton

### PR DESCRIPTION
**Pull request by samuelfreiberg**
### Platforms Impacted
- [ ] iOS
- [ ] macOS
- [X] win32 (Office)
- [X] windows
- [ ] android

### Description of changes

MenuButton has the prop accessibilityLabel, but it wasn't doing anything with it. With this change, we forward the accessibilityLabel to the inner Button component so that screen readers can properly read it.

### Verification

Manual with Narrator

### Pull request checklist

This PR has considered (when applicable):
- [ ] Automated Tests
- [ ] Documentation and examples
- [ ] Keyboard Accessibility
- [ ] Voiceover
- [ ] Internationalization and Right-to-left Layouts

### CODE REVIEW

**Comment by Saadnajmi** [comment]
Do we also need to do this for experimental menu button?
Also, do we need to do something like #1116 for menu button?

**Comment by ksiler** [comment]
> Do we also need to do this for experimental menu button?

Looks like!

> Also, do we need to do something like #1116 for menu button?

Looks like MenuButton is passing props like `disabled` to Button so #1116 should apply.

**Comment by samuelfreiberg** [comment]
> Do we also need to do this for experimental menu button? Also, do we need to do something like #1116 for menu button?

Good catch, adding now

**Review by ksiler — APPROVED** [review]

File: packages/components/MenuButton/src/MenuButton.tsx [hunk 1]
```diff
@@ -72,6 +72,7 @@ export const MenuButton = compose<MenuButtonType>({
       startIcon,
       componentRef: stdBtnRef,
       onClick: toggleShowContextualMenu,
+      accessibilityLabel,
```
Comment (Saadnajmi) [inline packages/components/MenuButton/src/MenuButton.tsx:75]: I wonder if we should just do something like ...props to avoid this happening in the future.

**Review by Saadnajmi — APPROVED** [review]

File: packages/components/MenuButton/src/MenuButton.tsx (hunk 1 above)
Comment (ksiler) [reply packages/components/MenuButton/src/MenuButton.tsx:75]: That's a good suggestion and I don't see any issues with it. I think I'd prefer that.

Comment (samuelfreiberg) [reply packages/components/MenuButton/src/MenuButton.tsx:75]: I'd agree. This should probably be the norm for our components moving forward. I'll change it

**Review by ValentinaKozlova — APPROVED** [review]

File: packages/components/MenuButton/src/MenuButton.tsx (hunk 1 above)
Comment (samuelfreiberg) [reply packages/components/MenuButton/src/MenuButton.tsx]: Changed in newest commit

**Pull request merged**